### PR TITLE
Prelim information for CFP; Adds CFP to navigation bar

### PR DESCRIPTION
**Pull request by annalorimer**

### CODE REVIEW

File: _pages/cfp.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+layout: page
+title: CFP
+subtitle: CFP
+permalink: /cfp/
+---
+
+Our CFP will open on August 14 2017!
+
+For updates, follow us on [Twitter](https://twitter.com/starconuw) or [Facebook](https://www.facebook.com/starconuw/). 
+
```
Comment (arshiamufti) [inline _pages/cfp.md]: mention the mailing list as well?

File: _config.yml [hunk 2]
```diff
@@ -35,6 +35,9 @@ navigation:
   - title: Sponsors
     url: /sponsors/
     desc: Cha-ching
+  - title: CFP
+    url: /cfp/
+    desc: Be one of our speakers
```
Comment (evykassirer) [inline _config.yml]: is this too many things in the nav now? wondering if we should just link to it from the speakers page

File: _pages/cfp.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: CFP
+subtitle: CFP
+permalink: /cfp/
+---
+
+Our CFP will open on August 14 2017!
```
Comment (evykassirer) [inline _pages/cfp.md]: can you explicitly say what CFP stands for? and also when the CFP closes so people can get an idea of the timeline

File: _config.yml (hunk 2 above)
Comment (annalorimer) [inline _config.yml]: I tested it and it looks fine. I think it should be in the nav bar.

File: _pages/cfp.md (hunk 1 above)
Comment (evykassirer) [inline _pages/cfp.md]: also that updates will be posted here once they're available

Comment (annalorimer) [inline _pages/cfp.md]: Could do. Twitter and FB just seemed more effective lol

File: _pages/cfp.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (evykassirer) [inline _pages/cfp.md]: this page should be linked to from the speakers page and home page where it's already mentioned (I think?)

File: _pages/cfp.md (hunk 1 above)
Comment (evykassirer) [inline _pages/cfp.md]: still pending?

File: _pages/cfp.md (hunk 4 above)
Comment (evykassirer) [inline _pages/cfp.md]: home page change still pending?

Comment (annalorimer) [inline _pages/cfp.md]: What home page change?

Comment (annalorimer) [inline _pages/cfp.md]: Nvm just saw it.

**Comment by arshiamufti** [comment]
When the CFP is actually up, we should update the home page with an announcement type thing. This looks good to me.

File: _pages/cfp.md (hunk 4 above)
Comment (evykassirer) [inline _pages/cfp.md]: still not linked in home page, will add it now

Comment (arshiamufti) [inline _pages/cfp.md]: i think we can wait until the cfp is actually live to link it in the home page?

Comment (evykassirer) [inline _pages/cfp.md]: the details are out of date there though now, and since we mention it I'd rather have up to date linking

**Pull request merged**